### DISCUSSION TITLE
Long-standing bug: going direct to a url with a bookmark hash does not work

### DISCUSSION THREAD

**Discussion by TAPgiles** [mdn/mdn-community#727] [Platform] [answered]
I've been using Chrome and using MDN for many years as a web developer. And I don't think this has _ever_ worked, honestly. At least not with the current skin/theme/version.

To see what I mean, click this link: [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_operators#comparison_operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_operators#comparison_operators) You're not taken to the relevant id within the page (the functionality built-in to browsers) even though the id exists, and clicking a link to that hash takes you there as expected.

And as someone who writes documentation for various platforms, linking to MDN resources--to the _bookmarks themselves_--is a really valuable way to help people get more info on some web-related topic. Only, it just doesn't really work.

Is this just a glaring bug no one has fixed for years, or am I missing something huge? This should just work, right?

**Comment by yin1999** [comment]
We have faced this problem for a long time (only happen on some of the pages in production). I believe the core dev team did notice this issue but has not yet fixed it. Some context:

- mdn/yari#5733
- mdn/yari#7450
- mdn/yari#8049

**Reply by bsmth** [reply, marked as answer]
I think the bug with the most relevant information is 

- [x] https://github.com/mdn/yari/issues/8049

As far as I can see, it looks like a fix is landing in Chrome 130:

* https://issues.chromium.org/issues/325081538#comment7
* https://chromestatus.com/feature/6270155647352832

**Reply by TAPgiles** [reply]
I see, thank you.

Honestly, even if they found that it's some bug in the browser itself, I'm surprised they never just put in a manual JS fix or something. Shouldn't be that much work to get it working so it doesn't go years without it working for users. That's another topic though I guess 😅

**Comment by pransh15** [comment]
Hi @TAPgiles 👋

Thank you for raising this concern. We have documented the issue and more instances mentioned regarding the broken links and the missing assets, as referenced in this [issue](https://github.com/mdn/yari/issues/7168) and a few others.

We have planned a platform update that will address these issues. This update is currently in the works, and we aim to resolve these issues as part of this broader improvement.

Thank you again for bringing this to our attention!

**Reply by TAPgiles** [reply]
Okay thanks 👍

By the way, I think there might be something wrong with this forum? I had the page open (or suspended) and would refresh multiple times a day. Only today did all these responses appear though--apparently from the day I posted!

**Comment by bsmth** [comment]
Looks like this is resolved now, see https://github.com/mdn/yari/issues/8049

Thanks, all!
